runtime/autoload/health/provider.vim

```diff
@@ -523,7 +523,8 @@ function! s:check_node() abort
   if empty(host)
     call health#report_warn('Missing "neovim" npm package.',
           \ ['Run in shell: npm install -g neovim',
-          \  'Is the npm bin directory in $PATH?'])
+          \  'Run in shell (if you use yarn): yarn global add neovim',
+          \  'Is the global node_modules/bin directory in $PATH?'])
```

Comment:
What does this mean? Either `node_modules/` or `bin/` is in `PATH`?

Reply:
Maybe that was not clear from my end (& also wrong). `node_modules` stores the `bin` executables inside `node_modules/.bin` folder that's how you can run these binaries either globally or not.

Reply:
The point of `npm root -g` when detecting the node.js host is that it doesn't use `node_modules/.bin` because those are shims. Shims don't work on Windows.

Reply:
Actually I think maybe this message can be removed? `'Is the global node_modules/bin directory in $PATH?'` 

Because:
1. `npm` will symlink the executables on mac or Linux in `/usr/local/bin` (don't know about windows)
2. `yarn` you will need to add `$(yarn global dir)/node_modules` in your `$PATH`

Reply:
I'd remove it. In Windows, npm will **copy** (ie. `npm link neovim` copies the `node_modules/neovim/` and its dependencies).

Reply:
@justinmk are you also ok with removing this line?

`'Is the global node_modules/bin directory in $PATH?'` or before my change `'Is the npm bin directory in $PATH?'`

Reply:
sure